Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -242,6 +242,83 @@ class PlaceDetailViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
 
+    func openCard(forPlace place: Place) {
+        if currentCardViewController.place == place {
+            currentCardViewController.place = place
```

Comment:
No redundant. The place that the card had before this function was called had no event. The place triggering this function has an event, therefore is necessary. I'll rename things to make that clearer